src/dbus/client/thread_api_dbus.hpp

```diff
@@ -693,6 +706,19 @@ class ThreadApiDBus
      */
     ClientError GetActiveDatasetTlvs(std::vector<uint8_t> &aDataset);
 
+    /**
+     * This method gets the active operational dataset
```

Comment:
Update the documentation? 
`This method gets the active operational dataset`

Reply:
Done.